corehq/messaging/scheduling/async_handlers.py

```diff
@@ -24,6 +29,49 @@ def recipients_response(self):
         ]
         return ret
 
+    @property
+    def user_group_recipients_response(self):
+        domain = self.request.domain
+        query = self.data.get('searchString')
+        group_result = (
+            GroupES()
+            .domain(domain)
+            .not_deleted()
+            .search_string_query(query, default_fields=['name'])
+            .size(10)
+            .sort('name')
+            .source(('_id', 'name'))
+        )
+        return [
+            {'id': group['_id'], 'text': group['name']}
+            for group in group_result.run().hits
+        ]
+
+    @property
+    def user_organization_recipients_response(self):
+        domain = self.request.domain
+        query = self.data.get('searchString')
+        result = (
+            SQLLocation
+            .active_objects
+            .filter(domain=domain, name__icontains=query)
```

Comment:
Manish wrote a location search that also searches with / https://github.com/dimagi/commcare-hq/pull/17958/ . So that you can search "MA / Boston" or something similar. Should this input allow that as well?

Also from that PR, I think you should be able to search for site code

Reply:
just looked into that, and it looks like it will require a bit of work to integrate with. since the location selection mechanism in the current broadcast ui is similar to what's implemented in this pr, i'd rather leave this as is for now and add that later as an enhancement if we choose to

Reply:
👍